docs/en/compute/AWS-EMR.md

```diff
@@ -0,0 +1,123 @@
+service.
+2. Make sure that the files that you downloaded are 
+3. Configure the below command with the required parameters. The root-ufs-uri should be an s3a:// or hdfs:// URI designating the root mount of the Alluxio file system.
+
+```bash
+aws emr create-cluster --release-label emr-5.23.0 --instance-count <num-instances> --instance-type <instance-type> --applications Name=Presto Name=Hive --name '<cluster-name>' --bootstrap-actions Path=s3://bucket/path/to/alluxio-emr.sh,Args=<web-download-url>,<root-ufs-uri>,<additional-properties> --configurations file:///path/to/file/alluxio-presto.json --ec2-attributes KeyName=<ec2-keypair-name>
+```
+
+3. On the [EMR Console](https://console.aws.amazon.com/elasticmapreduce/home), you should be able to see the cluster
+going through the different stages of setup. Once the cluster is in the 'Waiting' stage, click on the cluster details
+to get the 'Master public DNS'. SSH into this instance using the keypair provided in the previous command.
+4. Test that Alluxio is running as expected by running `sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"`
+
+Alluxio is installed in `/opt/alluxio/` by default. Hive and Presto are already configured to connect to Alluxio. The
+cluster also uses AWS Glue as the default metastore for both Presto and Hive. This will allow you to maintain table
+definitions between multiple runs of the Alluxio cluster.
+
+## Creating a Table
+
+The simplest step to using EMR with Alluxio is to create a table on Alluxio and query it using Presto/Hive.
+
+1. SSH into the 'hadoop' user in the master node. Then switch to the 'alluxio' user.
+2. Create a directory in Alluxio to be the external location of your table.
+```bash
+/opt/alluxio/bin/alluxio fs mkdir /testTable
+```
+3. Set the 'hadoop' user to be the owner of the directory
+```bash
+/opt/alluxio/bin/alluxio fs chown hadoop:hadoop /testTable
+```
+4. Exit to switch back into the 'hadoop' user and start the hive CLI.
+```bash
+hive
+```
+5. Create a new database to see if AWS Glue is working as expected. Check the [console](https://console.aws.amazon.com/glue/home)
+to see if the database is created.
+```sql
+CREATE DATABASE glue;
+```
+6. Use the newly created database and define a table.
+```sql
+USE glue;
+create external table test1 (userid INT,
+age INT,
+gender CHAR(1),
+occupation STRING,
+zipcode STRING)
+ROW FORMAT DELIMITED
+FIELDS TERMINATED BY '|'
+LOCATION 'alluxio:///testTable';
+```
+7. Create the Presto /tmp directory
+```bash
+#Create Presto temp directory
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio fs mkdir /tmp"
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio fs chmod 777 /tmp"
+presto-cli --catalog hive
+```
+8. Insert values into the table
+```sql
+USE glue;
+INSERT INTO test1 VALUES ('1', 24, 'F', 'Developer', '12345');
+```
+9. Read back the values in the table
+```sql
+SELECT * FROM test1;
+```
+
+##Customization
```

Comment:
a space is needed between # and section title for markdown to process it correctly